web/forms.py

```
@@ -1174,13 +1176,26 @@
     def __init__(self, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)